db/migrate/20230922102926_revoking_not_used_authorization_request_with_no_datapass.rb

```diff
@@ -0,0 +1,21 @@
+class RevokingNotUsedAuthorizationRequestWithNoDatapass < ActiveRecord::Migration[7.0]
+  def change
```

Comment:
```suggestion
  def up
```

faut faire un down aussi, quid ici ?

Reply:
J'ai invoqué la flemme, tu m'as remis dans le droit chemin

Reply:
On ne flemme pas avec l'intégrité de la données :}